vpi/quickstart/timestep/vpi.c

```diff
@@ -0,0 +1,73 @@
+#include<vpi_user.h>
+#include<inttypes.h>
+#include<stdio.h>
+#include "common_vpi.h"
+
+#define STOP_ITERATION 10
+
+uint32_t iteration = 0;
+
+/* In this example the simulation time is advanced from
+ * VPI. This is accomplished registering a chain of callbacks.
+ */
+
+// register a cbAfterDelay callback that executes delay_ro_cb after 1 timestep
```

Comment:
I think that some discussion/explanation is required about why four different callbacks are used. Why do we need to register `cbReadOnlySynch` if `cbReadWriteSynch` is going to be executed in the same chain anyway?